sw/device/tests/sim_dv/lc_walkthrough_test.c

```diff
@@ -81,31 +81,27 @@ bool test_main(void) {
     CHECK_DIF_OK(dif_otp_ctrl_is_digest_computed(
         &otp, kDifOtpCtrlPartitionSecret0, &secret0_locked));
 
-    if (secret0_locked == false) {
+    if (!secret0_locked) {
       // Write LC tokens to OTP secret0 partition.
       uint64_t otp_unlock_token_0 = 0;
       uint64_t otp_unlock_token_1 = 0;
       for (int i = 0; i < LC_TOKEN_SIZE; i++) {
         if (i < LC_TOKEN_SIZE / 2) {
-          otp_unlock_token_0 =
-              otp_unlock_token_0 | ((uint64_t)kOtpUnlockToken[i] << (i * 8));
+          otp_unlock_token_0 |= ((uint64_t)kOtpUnlockToken[i] << (i * 8);
```

Comment:
nit: is this a typo?

```suggestion
          otp_unlock_token_0 |= (uint64_t)kOtpUnlockToken[i] << (i * 8);
```

Reply:
Thanks for catching that, update it.